packages/@react-spectrum/tabs/stories/Tabs.stories.js

```diff
@@ -65,7 +65,7 @@ storiesOf('Tabs', module)
 
 function render(props = {}) {
   return (
-    <Tabs defaultSelectedKey="val1" {...props} onSelectionChange={action('onSelectionChange')}>
+    <Tabs {...props} maxWidth={500} onSelectionChange={action('onSelectionChange')}>
```

Comment:
did you want to apply this to the icon stories as well? currently they stretch the entire page
Also, should we use a spectrum value?

Reply:
Added.